community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -750,8 +748,8 @@ object projects:
 
   lazy val fs2 = SbtCommunityProject(
     project = "fs2",
-    sbtTestCommand = "coreJVM/test; coreJS/test",  // io/test requires JDK9+
-    sbtPublishCommand = "coreJVM/publishLocal; coreJS/publishLocal",
+    sbtTestCommand = """set coreJVM/scalacOptions -= "-Xfatal-warnings"; coreJVM/test; set coreJS/scalacOptions -= "-Xfatal-warnings"; coreJS/test""",  // io/test requires JDK9+
+    sbtPublishCommand = """set coreJVM/scalacOptions -= "-Xfatal-warnings"; coreJVM/publishLocal; set coreJS/scalacOptions -= "-Xfatal-warnings"; coreJS/publishLocal""",
```

Comment:
If you wish :)

```suggestion
    sbtTestCommand = "set ThisBuild/tlFatalWarnings := false; coreJVM/test; coreJS/test",  // io/test requires JDK9+
    sbtPublishCommand = "set ThisBuild/tlFatalWarnings := false; coreJVM/publishLocal; coreJS/publishLocal",
```

Reply:
#14407 has now been fixed so we can re-enable fatal-warnings :)